.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
+    "extends": [
+        "airbnb"
+    ],
+    "plugins": [
+        "react"
+    ],
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true,
+            "impliedStrict": true
+        },
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {
+        "quotes": 0,
+        "new-cap": 0
+    },
+    "env": {
+        "browser": true,
+        "es6": true,
```

Comment:
If the ES6 is set to be true here, why is hound giving issues?

Reply:
Vicky that will be a problem fixed if hound is integrated with our project.
And that's a task Linc is working on now.